### ISSUE TITLE
Not working for me

### ISSUE THREAD

**Issue by explorador** [UseAllFive/base-tumblr#4]
Grunt it's giving me errors of missing files, I can't do anything...

**Comment by casconed** [comment]
Paste the error here. "Missing files" doesn't tell us anything. 






On Sat, Apr 11, 2015 at 2:22 PM -0700, "explorador" <notifications@github.com> wrote:












Grunt it's giving me errors of missing files, I can't do anything...

**Comment by explorador** [comment]
You are right, my bad. This is the error:
```
Running "sass:build" (sass) task
Error: File to import not found or unreadable: _normalize.
       Load paths:
         /Users/Explorador/Sites/tumblr/Test/web/css
         /Users/Explorador/Sites/tumblr/Test/web/vendor
         /Users/Explorador/Sites/tumblr/Test/web/vendor/bootstrap-sass-official/vendor/assets/stylesheets/bootstrap
         /Users/Explorador/Sites/tumblr/Test/web/vendor/bourbon/dist
        on line 1 of web/css/application.scss
  Use --trace for backtrace.
Warning: Exited with error code 65 Use --force to continue.

Aborted due to warnings.
```

**Comment by casconed** [comment]
I can't reproduce. `normalize` should get installed as part of `bower
install`. Did you have any errors running `npm install` or `bower install`?

**Comment by casconed** [comment]
I can't reproduce. `normalize` should get installed as part of `bower
install`. Did you have any errors running `npm install` or `bower install`?

**Comment by explorador** [comment]
with npm install only this warning: 
```npm WARN deprecated grunt-jscs-checker@0.6.2: Package was renamed to 'grunt-jscs```

With bower this:
```bower invalid-meta  twitter is missing "ignore" entry in bower.json
bower invalid-meta  facebook is missing "ignore" entry in bower.json
bower invalid-meta  Backbone.GroupedCollection is missing "main" entry in bower.json
bower invalid-meta  Backbone.GroupedCollection is missing "ignore" entry in bower.json
bower invalid-meta  requirejs-handlebars is missing "main" entry in bower.json
bower invalid-meta  requirejs-handlebars is missing "ignore" entry in bower.json
```

And this note with bower:
```
Please note that,
    backbone.marionette#2.0.3 depends on jquery#1.8.0 - 2.1.0 which resolved to jquery#2.1.0
    UseAllFive/base-marionette depends on jquery#~2.1.1 which resolved to jquery#2.1.3
```

**Comment by casconed** [comment]
What is the output of `sass -v`? You may also have to run `bundle install` if you are using an older version of sass.

**Comment by explorador** [comment]
Sass 3.4.13 (Selective Steve)

**Comment by casconed** [comment]
Hmm me too. Trying running `bundle install`? Also, do you see `normalize-css` in your `web/vendor` directory?

**Comment by explorador** [comment]
Y tried bundle install and didn't work. I don't see the vendor directory under web, only css, img, js, rendered and a file called index.template

**Comment by casconed** [comment]
do you have a file called `.bowerrc`? Also, what version of bower are you running? When you run `bower install` it should install those deps in `web/vendor`

**Comment by explorador** [comment]
Ha! I didn't have that file. I had to show hidden files on mac and then download the repo again. Now doesn't have errors. Thanks man! Just one question. When I modify the sass files is not reflecting the changes, I'm viewing the site in http://localhost:8010/  Doesn't show any error. also the same thing for index.template. I can't see what I modified.

**Comment by casconed** [comment]
If you leave the grunt watch task running, your sass changes should get recompiled.

**Comment by explorador** [comment]
Yeah, the main.css is changing but I can't see the changes reflected on localhost:8010. Also when I modify index.template the index.html in the render directory is not changing... I even tried restarting the node.js server

**Comment by casconed** [comment]
I'm not 100% certain but I don't think index.template is meant to be modified - that is what you paste into your tumblr to be able to use marionette with a tumblr theme. the "rendered" output is what you would see as output if you served that index.template from tumblr. 

The local server is meant to test javascript changes. If you want to add css you'll have to change the output destination in the grunt sass task and link the main.css in `rendered/index.html` - no local css is loaded, currently.
